pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -219,6 +226,11 @@ func (s *discovery) getServiceInfo(proc *process.Process) (*serviceInfo, error)
 		return nil, err
 	}
 
+	createTime, err := proc.CreateTime()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
That is out of scope for the PR as the gap exists from day 1, but getting the creation time of a process without timezone information, is going to be problematic for customers
I think we need to re-work the creation time and gain all information we need to properly show it to customers

cc @LotharSee